### ISSUE TITLE
Clean mess with ViewAttributeElement priority.

### ISSUE THREAD

**Issue by scofalik** [ckeditor/ckeditor5-engine#378]
There is some mess with `ViewAttributeElement` priority, especially in `model-to-view-converters.js` and `ModelConverterBuilder`. Right now, event priority is always used to determine priority for `engine.treeView.Writer#wrap` and `#unwrap`. It should acknowledge `ViewAttributeElement` priority and use event priority only if the former is not set.

**Comment by pjasiun** [comment]
I means that the `AttributeElement` priority is ignored and always overwritten by event priority. Also if nothing is set default priority is a event default priority, not an attribute default priority. On the other hand we want these priorities to be the same in most cases.

**Comment by oleq** [comment]
I stumbled upon this issue when working on https://github.com/ckeditor/ckeditor5-link/issues/121. The event priority set by `withPriority()` looked like the right solution but later on, thanks to @pjasiun, I found out I needed to customize the `LinkElement#priority` to get things working. TBH, it's confusing what is what and which is which, especially that the first one has a nice chainable syntax and the second one feels more like a hack.
